src/irmin-pack/unix/control_file_intf.ml

```diff
@@ -152,12 +162,20 @@ module Payload_v4 = struct
   type t = {
     dict_end_poff : int63;
     suffix_end_poff : int63;
-    status : status;
     upgraded_from_v3_to_v4 : bool;
+    chunk_start_idx : int;
```

Comment:
`chunk_start_idx` is replacing `generation`, right? Is the plan to eventually remove the `generation` field?

Reply:
The two will be the same for a period of time when existing stores migrate to V4, but eventually they will move independently and `generation` will still be needed to facilitate GC swap of prefix and mapping files.

Reply:
I see, thanks